### PR TITLE
Knee Osteoarthritis Prediction

### PR DESCRIPTION
**Pull request by devarshh08**
### Description
A clear and concise description of what the PR does.

This model predicts whether the patient has knee osteoarthritis based on xray daya divided into 5 severity grades. And the prediction cell compares the given xray to the trained model and predicts how severe the osteoarthritis is.

### Related Issues
Fixes issue #168 

### Changes
List the detailed changes made in this PR.
 - Made a model on Knee Osteoarthritis prediction using CNN
 - Usage of libraries: Keras, Numpy, Matplotlib, Tensorflow
 - It has an accuracy of 94.15% and a loss of 15.68%

### Checklist
Make sure to check off all the items before submitting. Mark with [x] if done.
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I am working on this issue under GSSOC

### CODE REVIEW

**Comment by devarshh08** [comment]
@SrijanShovit Please look into the PR sir.

**Comment by devarshh08** [comment]
@SrijanShovit Please look into it sir.
